x-pack/plugins/visualization_editor/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "author": "Elastic",
+  "name": "visualization_editor",
+  "version": "7.0.0",
```

Comment:
Should this be 8.0? Not sure about how these sub packages work

Reply:
I don't know, there's only one example of a package.json in an x-pack plugin, and it has `7.0.0`

Reply:
doesn't matter; the version is arbitrary i believe (we strangely have `"version": "kibana"` specified in many of the `core_plugins`), and these package files will be changed by the time we get to the new platform anyway.